### ISSUE TITLE
We need a refactor

### ISSUE THREAD

**Issue by vakowal** [vakowal/sda-regional-pathways#37]
Things are getting cluttered in 02_analysis_and_plots.R and in our figures folder. Can we streamline and simplify?

**Comment by vakowal** [comment]
wondering if it might be easier to organize our code by adding separate scripts for discrete analysis or figure plotting tasks that likely don't need to be replicated in sequence. I started with this strategy by adding a new script today for the CRREM SDA pathways task (described in PR #40 ).

**Comment by e-feth** [comment]
Agreed that things are getting a little cluttered - maybe we can focus on this once we get through final submission of CRREM comments this week? Don't want to accidentally break something when we might need to do some fast turnaround analysis to finish up comments.

**Comment by vakowal** [comment]
that plan sounds good!
